aws/aws-event-streams/build.gradle.kts

```diff
@@ -9,5 +9,6 @@ extra["moduleName"] = "software.amazon.smithy.java.aws.events"
 
 dependencies {
     api(project(":core"))
+    api(project(":http:http-api"))
```

Comment:
Is this needed? I glanced but didn't see HTTP things actually getting used

Reply:
> Is this needed? I glanced but didn't see HTTP things actually getting used

The RPC response util class has a method that takes a `HttpResponse` instance to create the API response. I will remove this dependency and the caller will need to create and pass down the data-stream.